metrics/metrics.go

```diff
@@ -47,6 +47,9 @@ const (
 	labelGrpcStatusCode   = "grpc_status_code"
 	unknownCSIDriverName  = "unknown-driver"
 
+	// LabelMigrated is the Label that indicate whether this is a migrated operation
```

Comment:
maybe call it "CSI migration" so it's more obvious what "migrated" means

Reply:
Done